### PR TITLE
[Calls] Added changelog file and entries

### PR DESCRIPTION
**Pull request by dcpena**
Added changelog file and entries to address PCX-13087.

### CODE REVIEW

File: src/content/changelogs/calls.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+link: "/calls/changelog/"
+productName: Calls
+productLink: "/calls/"
+productArea: Developer platform
+productAreaLink: /workers/platform/changelog/platform/
+entries:
+  - publish_date: "2024-09-25"
+    title: TURN service is generally available (GA)
+    description: |-
+      Cloudflare Calls TURN service is generally available and helps address common challenges with real-time communication such connectivity, privacy, performance, and scalability. For more information, refer to the [blog post](https://blog.cloudflare.com/webrtc-turn-using-anycast/) or [TURN documentation](/calls/turn/).
```
Comment (nils-ohlmeier) [inline src/content/changelogs/calls.yaml]: I would prefer if we limit the list of attributes to "such as connectivity and privacy".

It depends on what we are attributing here. Any real-time communication which works without TURN has better performance. If we compare our TURN service to other TURN services, then yes our scales better and should be better in performance. But that distinction is not very clear to me here. Thus I would suggest to be a little bit more humble.

Comment (renandincer) [reply src/content/changelogs/calls.yaml]:
```suggestion
      Cloudflare Calls TURN service is generally available and helps address common challenges with real-time communication. For more information, refer to the [blog post](https://blog.cloudflare.com/webrtc-turn-using-anycast/) or [TURN documentation](/calls/turn/).
```

File: src/content/changelogs/calls.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - publish_date: "2024-04-04"
+    title: Cloudflare Calls open beta and Orange Meets availability
```
Comment (renandincer) [inline src/content/changelogs/calls.yaml]: Should these be two separate points on the same day?

Comment (dcpena) [reply src/content/changelogs/calls.yaml]: Done!

**Review by nils-ohlmeier — APPROVED** [review]

**Review by patriciasantaana — APPROVED** [review]

File: src/content/changelogs/calls.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    description: |-
+      Cloudflare Calls TURN service is generally available and helps address common challenges with real-time communication. For more information, refer to the [blog post](https://blog.cloudflare.com/webrtc-turn-using-anycast/) or [TURN documentation](/calls/turn/).
+  - publish_date: "2024-04-04"
+    title: Orange Meets availability
+    description: |-
+      Orange Meets, Cloudflare's internal video conferencing app, is open source and available for use from [Github](https://github.com/cloudflare/orange?cf_target_id=40DF7321015C5928F9359DD01303E8C2). 
+  - publish_date: "2024-04-04"
+    title: Cloudflare Calls open beta
+    description: |-
+      Cloudflare Calls is in open beta and available from the Cloudflare Dashboard.
```
Comment (patriciasantaana) [inline src/content/changelogs/calls.yaml:19]:
```suggestion
      Cloudflare Calls is in open beta and available from the Cloudflare dashboard.
```

**Pull request merged**